### ISSUE TITLE
text to speech

### ISSUE THREAD

**Issue by prairitm** [JuzSer/vue-bot-ui#12]
is there any functionality where we can have speech for the texts on the ui as it replies?

**Comment by juzser** [comment]
@prairitm Do you mean accessibility for blind people? 
I listed accessibility in feature list but it's a huge task, and it cannot come soon.

**Comment by juzser** [comment]
Inactivity
